src/lib/components/glue/Modal.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+	/**
+	 * @type {boolean}
+	 */
+	export let showModal: boolean;
```

Comment:
we don't really need doc strings because typescript `: boolean` already denotes type

Reply:
fixed